manuscript.Rmd

```diff
@@ -147,7 +142,7 @@ On the contrary, a feature of good software is modularity because it enables reu
 ## ID 7: Contributor Roles and Involvement
 
 We support the standardized assessment of project contributor roles, similar to the Contributor Roles Taxonomy (CRediT; https://credit.niso.org). 
-However, we like to point out that the current evaluation schemes yields lower scores for the same effort of the individual researcher if they are part of a larger software project.
+However, we like to point out that the current evaluation schemes yields different scores for the same effort of the individual researcher depending on the size of the software project.
```

Comment:
Well, it is (size of the project)*(contribution as a fraction of the whole project). I think in general this is okayish. I think that maybe we don't describe the problem very well, because I believe the problem is that "size of the project" is a bad indicator overall, and therefore also the multiplication is a bad indicator. I think the real (but difficult) question is how to score the individual contribution if we have "rigor" as the main criterion - for example, what if one researcher writes the code and the other researcher writes documentation and test - who gets the rigor points?

I think your suggested change is good and I would leave it like that for now; we can't do more than just noting the problem exists at this point.